cachi2/core/package_managers/rpm/main.py

```diff
@@ -28,6 +29,36 @@ def fetch_rpm_source(request: Request) -> RequestOutput:
                 solution=("Check correct 'yaml' syntax in the lockfile."),
             )
 
+        redhat_rpms_lock = RedhatRpmsLock(raw_content)
+        log.debug("Checking format '{}'".format(redhat_rpms_lock.__class__.__name__))
+        if not redhat_rpms_lock.match_format():
+            raise PackageRejected(
+                f"Rpm lockfile '{DEFAULT_LOCKFILE_NAME}' format is not supported'.",
+                solution=("Check 'lockfileVendor' and 'lockfileVersion' keys in the lockfile."),
+            )
+
+        log.debug("Lockfile content validation")
```

Comment:
Disregard this, with my latest comments this will be completely superseded by my suggestion in: https://github.com/containerbuildsystem/cachi2/pull/502#discussion_r1554154284